content/applications/hr/payroll.rst

```diff
@@ -108,6 +108,16 @@ The contract template forms the basis of an offer, and can be modified for speci
 employees, when necessary. If a contract template is created or modified in the *Payroll*
 application, the changes are also reflected in the *Recruitment* application.
 
+.. important::
+   To access contract templates, the *Salary Configurator* module **must** be installed.
+
+   To install this module, navigate to the :menuselection:`Apps` application. In the search bar,
+   delete the :guilabel:`Apps` filter, then type in `Salary Configurator`.
+
+   Click :guilabel:`Install` on the *Salary Configurator* module. Then, that module, along with the
+   *Salary Configurator - Holidays* and *Salary Configurator - Payroll* modules install, as well.
+   Once the modules are installed, the database reverts to the main dashboard.
+
```

Comment:
No need to explain how to install the module, since there is documentation that already exists for that :) also can specify the technical name as well for added clarity 
```suggestion
.. important::
   To access contract templates, the *Salary Configurator* (`hr_contract_salary`) module **must** be
   :ref:`installed <general/install>`.
```